src/main.py

```diff
@@ -141,6 +145,9 @@ def load_games_from_disk(self):
                 game = Game(data)
                 shared.store.add_game(game, {"skip_save": True})
 
+    def get_source_name(self, source_id):
+        return globals()[f"{source_id.title()}Source"].name
```

Comment:
`source_id` can be different from the source class name:
```
Traceback (most recent call last):
  File "/app/share/cartridges/src/importer/importer.py", line 206, in pipeline_advanced_callback
    self.progress_changed_callback()
  File "/app/share/cartridges/src/importer/importer.py", line 219, in progress_changed_callback
    self.import_callback()
  File "/app/share/cartridges/src/importer/importer.py", line 229, in import_callback
    shared.win.create_source_rows()
  File "/app/share/cartridges/src/window.py", line 135, in create_source_rows
    label=self.get_application().get_source_name(source_id),
          ^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^
  File "/app/share/cartridges/src/main.py", line 149, in get_source_name
    return globals()[f"{source_id.title()}Source"].name
           ~~~~~~~~~^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^
KeyError: 'Lutris_WineSource'
```

We should avoid using `globals`, `getattr` and `setattr`, they can't work with linters and make bugs harder to catch. If feel the need for them, we should instead tweak the underlying structure to fit the new use case.

Reply:
This is the easiest way of doing stuff like this for now and I didn't want to touch your code.

It works for now, but if you have better ideas, we can discuss later.